xds/internal/client/client_test.go

```diff
@@ -32,20 +31,25 @@ import (
 	corepb "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
 )
 
-const balancerName = "dummyBalancer"
-
-var validConfig = bootstrap.Config{
-	BalancerName: balancerName,
-	Creds:        grpc.WithInsecure(),
-	NodeProto:    &corepb.Node{},
+func clientOpts(balancerName string) Options {
+	return Options{
+		Config: bootstrap.Config{
+			BalancerName: balancerName,
+			Creds:        grpc.WithInsecure(),
+			NodeProto:    &corepb.Node{},
+		},
+		DialOpts: []grpc.DialOption{grpc.WithBlock()},
```

Comment:
Add `WithTimeout()` (this seems to be the easiest test though it's deprecated)

Reply:
Done.